docs/cli.md

```diff
@@ -0,0 +1,72 @@
+# OpenWhisk CLI
+
+OpenWhisk offers a powerfull command line interface that allows complete management of all aspects of the system.
+
+## Setting up the OpenWhisk CLI 
+
+- Building OpenWhisk from a cloned repository will result in the generation of the command line interface. The
```

Comment:
verb tense: will result, will be, will run, then are located. I'd prefer present tense everywhere.